### DISCUSSION TITLE
Localite Trigger Marker Import?

### DISCUSSION THREAD

**Discussion by ls034** [simnibs/simnibs#383] [8. Simulation workflows]
I'm trying to upload TMS coil positions and angles obtained as trigger markers (.xml files) in Localite to simnibs to run a simulation from > 20,000 pulses on one subject. I see the tutorial [here]((https://simnibs.github.io/simnibs/build/html/documentation/neuronavigation/localite.html)), but I'm confused how to get from the output of tms_list into simnibs and run a e-field simulation from all of those pulses. Is there a way to upload the tms_list file under TMS pos list in the GUI? Any help would be greatly appreciated!

**Comment by Kstoffer** [comment]
We do not readily support import in the GUI, however, it should be straightforward to loop over the positions using a combination of the reading script in:
https://simnibs.github.io/simnibs/build/html/documentation/neuronavigation/localite.html
and the scripting of simulations in:
https://simnibs.github.io/simnibs/build/html/tutorial/scripting.html
Best,
Kristoffer
